conjure-java-client-verifier/src/test/java/com/palantir/verification/Cases.java

```diff
@@ -31,23 +31,18 @@
 
 public final class Cases {
 
-    public static final ClientTestCases TEST_CASES = deserializeTestCases(
-            new File("build/test-cases/test-cases.json"));
-    private static final IgnoredClientTestCases IGNORED_TEST_CASES = deserializeIgnoredClientTestCases(
-            new File("src/test/resources/ignored-test-cases.yml"));
+    public static final ClientTestCases TEST_CASES = deserializeTestCases(new File("build/test-cases/test-cases.json"));
+    private static final IgnoredClientTestCases IGNORED_TEST_CASES =
+            deserializeIgnoredClientTestCases(new File("src/test/resources/ignored-test-cases.yml"));
 
     private Cases() {}
 
     private static ClientTestCases deserializeTestCases(File file) {
         try {
-            return new ObjectMapper()
-                    .registerModule(new Jdk8Module())
-                    .readValue(file, TestCases.class)
-                    .getClient();
```

Comment:
I actually prefer the old formatting here - I think I find it easier to scan the method names, don't really _need_ it to be a one-liner

Reply:
Fixed by 0.3.2